machine.go

```diff
@@ -619,10 +626,32 @@ func (m *Machine) setupLogging(ctx context.Context) error {
 		return err
 	}
 
-	m.logger.Debugf("Configured VMM logging to %s, metrics to %s",
-		m.Cfg.LogFifo,
-		m.Cfg.MetricsFifo,
-	)
+	m.logger.Debugf("Configured VMM logging to %s", path)
+
+	return nil
+}
+
+func (m *Machine) setupMetrics(ctx context.Context) error {
+	fmt.Println("setupMetrics")
```

Comment:
m.logger.printf if we still want this message?

Reply:
Right. Will fix.